### ISSUE TITLE
code_blocks => false in ~/.mdlrc

### ISSUE THREAD

**Issue by devth** [markdownlint/markdownlint#166]
How do we customize rules in `~/.mdlrc`?

This does not appear to work:

```ruby
rule 'MD013', :code_blocks => false, :tables => false
```

mdl still reports "MD013 Line length" on code blocks with lines > 80 chars.

**Comment by waylan** [comment]
I'd like to see this also.

I realize you can define rulesets in style files and then call that style in your `.mdlrc` file. But I don't need multiple rulesets. A single config in `.mdlrc` is all I need. That file should be able to config all aspects of the configuration, including a custom ruleset/style.

**Comment by mnarrell** [comment]
Where do I put custom style `.rb` definitions?  What does an entry in the `.mdlrc` look like when referencing custom styles?

**Comment by shadiakiki1986** [comment]
Anyone figure this out yet? The doc page [creating styles/parameters](https://github.com/markdownlint/markdownlint/blob/master/docs/creating_styles.md#parameters) has the same content of `.mdlrc` as @devth above, but `mdl` seems to be ignoring these overridden style parameters

**Comment by shadiakiki1986** [comment]
For my purposes, I just patched the default value of `MD007/indent` parameter in `/var/lib/gems/2.5.0/gems/mdl-0.5.0/lib/mdl/rules.rb` from 2 to 4

**Comment by barnett** [comment]
Seeing this today with the below being added but the parameter being ignored when running.
```
rule 'MD029', :style => :ordered
```

**Comment by jaymzh** [comment]
sorry missed this.

That is not a config, that's a style. Put that in a style file (call it anything, it doesn't matter, `mdl_style.rb` is common), and then call mdl with `-s mdl_style.rb` *or* add `style "mdl_style.rb"` to your `~/.mdlrc`.

Styles go in style files. You can have many styles.
You pick a style in your config file.
